### DISCUSSION TITLE
Should we release 0.72.8?

### DISCUSSION THREAD

**Discussion by fortmarek** [reactwg/react-native-releases#93] [Patches]
## Should we release 0.72.8?

Current Release: [0.72.7](https://github.com/facebook/react-native/releases/tag/v0.72.7)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### List of qualified picks

1. https://github.com/reactwg/react-native-releases/discussions/91#discussioncomment-7556866 ✅ 
1. https://github.com/facebook/react-native/commit/c5e7cd4ad96f468e9c5e2966e5fa5d599daa3e00 ✅ 
1. https://github.com/facebook/react-native/commit/cd56347dca4e948f5038643bcd804c41f037727a: https://github.com/facebook/react-native/pull/41989 ✅ 
1. https://github.com/facebook/react-native/commit/479b4bb1ba504706613f71530a4ddcf86ff1bcd9 ✅ 

#### Local commits to backport to main

1.

**Comment by vksgautam1986** [comment]
https://github.com/react-native-community/discussions-and-proposals/issues/725#issuecomment-1780222618. can we have this ?

**Reply by liamjones** [reply]
It's already listed above as item 2 in the list of qualified picks

**Reply by cortinico** [reply]
We discussed this a bit and we decided to don't include a major version bump in a patch release.
If you want to use Fresco 3.0.0 you can **either**:
1. Update RN to 0.73.0
2. Include the following lines in your `android/app/build.gradle`

```diff 
...
dependencies {
    ...
+   implementation("com.facebook.fresco:fresco:3.1.3")
+   implementation("com.facebook.fresco:middleware:3.1.3")
+   implementation("com.facebook.fresco:imagepipeline-okhttp3:3.1.3")
+   implementation("com.facebook.fresco:ui-common:3.1.3")
}
```

**Comment by BasixKOR** [comment]
Could we pick this too? https://github.com/facebook/react-native/commit/c5e7cd4ad96f468e9c5e2966e5fa5d599daa3e00

I think not calling `invalidate` on ViewManager's are just a bug and should be backported into all supported versions.

**Reply by fortmarek** [reply]
Picked ✅

**Comment by joe-sam** [comment]
Having LogBox crashes leads to extremely poor trace and debug for developer experience. 
 All LogBox fixes should immediately be backported to all "active supported" branches.
 This commit will land in 0.73, but has been included in 0.71 stable branch already.
 
[Prevent LogBox from crashing on long messages](https://github.com/facebook/react-native/pull/38005)

**Reply by fortmarek** [reply]
Picked ✅

**Comment by joe-sam** [comment]
Symbolicate unhandled promise rejections #40914 
https://github.com/facebook/react-native/pull/40914

[Symbolicate unhandled promise rejections](https://github.com/facebook/react-native/commit/18c9797ecc8dab220ec68aae011ddb73db17a99d)
 I think this might have been backported already to 0.72 stable but not made it into a release version yet ( but an RN build from original sources might just work ).

**Reply by joe-sam** [reply]
Ignore this pick request as it has already been incorporated in 0.72.7 via  #41377.

**Comment by tido64** [comment]
Hi folks, just wanted to check in and see if you're planning to publish 0.72.8 before the end of the year. I'm going to log off for the rest of the year after today and just wanted to make sure that these two PRs against the `0.72-stable` branch get merged while I'm off:

- https://github.com/facebook/react-native/pull/41445
- https://github.com/facebook/react-native/pull/41456

**Reply by fortmarek** [reply]
Both of them merged, thanks 🙌

**Comment by cipolleschi** [comment]
CircleCI got broken yesterday due to Ruby and bundler. This is the fix:
https://github.com/facebook/react-native/commit/479b4bb1ba504706613f71530a4ddcf86ff1bcd9

**Reply by lunaleaps** [reply]
Do we need this on all the release branches as well?

**Reply by fortmarek** [reply]
I believe so, as the change was on the CircleCI 🤔

**Reply by fortmarek** [reply]
This was picked as part of: https://github.com/facebook/react-native/pull/41989
